gitbook-docs/security.md

```diff
@@ -112,6 +126,92 @@ Clients using other kinds of protocols can include the `token` in the Signal K m
 }
 ```
 
+### Token Validation
+
+Tokens may have a short expiry time and need to be renewed periodically, or a token's current validity may be unknown. 
+
+#### Web Clients
+To validate a token, a web based client should send an HTTP GET request to `/signalk/«version»/auth/validate` with the token in the cookie.
+If the token is valid, it will be renewed, and the new cookie set in the response.
```

Comment:
_it will be renewed_ sounds like the existing token will be renewed.

Reply:
Changed